### PR TITLE
Create a new java class Test.java

### PR DESCRIPTION
**Pull request by cszddx**
[This change is from cszddx]

### CODE REVIEW

File: Test.java [hunk 1]
```diff
@@ -0,0 +1,3 @@
+public class Test {
+
```
Comment (DuAlex) [inline Test.java:2]: no method implementation. please finish it.

**Review by DuAlex — COMMENTED** [review]
please see the details.

File: Test.java (hunk 1 above)
Comment (cszddx) [reply Test.java:2]: Thanks for your reply. I will finish it later.

**Comment by DuAlex** [comment]
Look good to me.:)

**Pull request merged**